IM_calculation/scripts/calculate_ims.py

```diff
@@ -163,19 +191,30 @@ def main():
     if "FAS" in im:
         im_options["FAS"] = calc.validate_fas_frequency(args.fas_frequency)
 
+    advanced_im_config = advanced_IM_factory.advanced_im_config(
+        args.advanced_ims, args.advanced_im_config, args.OpenSees_path
+    )
+
```

Comment:
Move this down to the 
if args.adv_ims != None
And add an adv_im_cfg=None to the else condition